ui/components/show/showCards/ShowCards.tsx

```diff
@@ -12,72 +11,74 @@ import CardImgContainer from './ImgContainer/CardImgContainer';
 import TagsContainer from '../../layout/cardElements/TagsContainer/TagsContainer';
 import ContentSection from '../../layout/ContentSection/contentSection';
 
-import styles from "./ShowCards.module.scss";
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/swiper-bundle.min.css';
 
-interface ShowCardsProps{
-    shows: Show[]
-  }
+import styles from "./ShowCards.module.scss";
 
+interface ShowCardsProps {
+  shows: Show[];
+  singleRow?: boolean;  // New prop to control single row display
+}
 
-function ShowCards({shows}:ShowCardsProps) {
+function ShowCards({ shows, singleRow = false }: ShowCardsProps) {
+  if (!shows) {
+    return null;
+  }
 
+  const cardContent = (show: Show) => (
+    <Card key={show.id}>
+      <div>
+        <Heading>
+          <div className={styles.eperDate}>
+            <EpDate dtStr={show.newestEpDate} />
+          </div>
+          <ShowLink slug={show.slug}>
+            <h3 className={styles.title}>{show.showName}</h3>
+          </ShowLink>
+        </Heading>
+        <GradientBg colours={show.dom_colours}>
+          <ShowLink slug={show.slug}>
+            <CardImgContainer show={show} />
+          </ShowLink>
+        </GradientBg>
+      </div>
+      <ContentSection heading='Subscribe:' tag='h5' centered={true} spacing='tight'>
+        <FeedLinks showId={show.id} slug={show.slug} showName={show.showName} extFeeds={show.extFeeds} />
+      </ContentSection>
+      <div className={styles.desc}>
+        <Desc desc={show.desc} approxLength={125} />
+        <span>
+          <ShowLink slug={show.slug}>
+            &nbsp;More.
+          </ShowLink>
+        </span>
+      </div>
+      {show.tagIds && <TagsContainer currentTagIds={JSON.parse(show.tagIds)} maxTags={3} />}
+    </Card>
+  );
 
-    if (!shows){
-      return null;
-    } 
-    // console.log(shows[0]);
- 
-    return (
-      <>
+  return (
+    <>
+      {singleRow ? (
+        <Swiper
+          spaceBetween={10}
+          slidesPerView={'auto'}
```

Comment:
Particularly on narrow screens I think it would make it clearer that there is more horizontal content if we make sure people can see some of the next card.

```suggestion
          slidesPerView={1.1}
          breakpoints={{
            400: {slidesPerView: 'auto'}
          }}
```